jest.setup.js

```
@@ -0,0 +1,92 @@
+// Mock react-native-vector-icons
+jest.mock('react-native-vector-icons/FontAwesome6', () => 'Icon');
```

ESLint

[error] 'jest' is not defined.

(no-undef)